skuba-update/skuba_update/skuba_update.py

```diff
@@ -244,7 +244,7 @@ def restart_services():
     for service in result.output.splitlines():
         cmd = run_command(['systemctl', 'restart', service], needsOutput=False)
         if cmd.returncode != 0:
-            log('Warning! Service \'{0}\' restart returned non zero exit code')
+            log('Warning! Service \'{0}\' restart returned non zero exit code'.format(service))
```

Comment:
Starting with Python 3.6, "f-strings" are available which are faster than `string.format` and IMO easier to read.  I'd also prefer to switch to double quotes to avoid the escapes.  With that said, I wouldn't go back and change the code a second time just for this - I'm just sharing for the sake of future knowledge. :)
```suggestion
            log(f"Warning! Service '{service}' restart returned non zero exit code")
```

Reply:
I also prefer the f-string format. But I was just following the pattern already in the file. I`ll update both call then. ;)

Reply:
Done